index.html

```diff
@@ -57,6 +57,174 @@ <h2>I`am a Software Developer</h2>
+              </ul>
+            </div>
+            <div class="work2-button"><button type="button">See Project</button></div>
+        </div>
+        <div class="work-2">
+          <div class="work2-content">
+            <h1>Profesional Art Printing Data</h1>
+            <h2>Profesional Art Printing Data</h2>
+            <p>First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+            <ul><li class="list2">Html</li>
+              <li class="list2">Bootstrap</li>
+              <li class="list2">Ruby </li>
+            </ul>
+          </div>
+          <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+      <div class="work-2">
+        <div class="work2-content">
+          <h1>Website Protfolio</h1>
+          <h2>Website Protfolio</h2>
+          <p>Sixth A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard</p>
+          <ul>
+            <li class="list2">Html</li>
+            <li class="list2">Bootstrap</li>
+            <li class="list2">Ruby </li>
+          </ul>
+        </div>
+        <div class="work2-button"><button type="button">See Project</button></div>
+      </div>
+        </div>
+          <div class="popUp-Container"></div>
+        </section>
+        <section class="about-me-section" id="about">
+          <div class="about-me-1">
+            <div class="about-me1-content">
+              <h1>About me</h1>
+              <p>
+                Hello I`m a software developer!  I can help you build a product ,
+                feature or website Look through some of my work and experience! If
+                you like what you see and have a project you need coded, don`t
+                hestiate to contact me.
+              </p>
+              <div class="about-me-btn"><input type="button" class="about-me-button" value="Get my Resume"></div>
+            </div>
+            <div class="about-me-background" alt="Backgroud Image"></div>
+          </div>
+          <div class="about-me1-separation">
+            <div class="line2"></div>
+          </div>
+          <ul class="about-me-2">
+            <li class="grid1">
+              <div class="topic1">
+                <img src="./svg/language.svg" alt="Language" />
+                <p>Language</p>
+              </div>
+              <ul class="discussion">
+                <li><a href="https://en.wikipedia.org/wiki/JavaScript">Javascript</a></li>
+                <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+              </ul>
+            </li>
+            <li class="grid2">
+              <div class="topic2">
+                <img src="./svg/frameworks.svg" alt="Framework" />
```

Comment:
- [ ] As the previous review requested,  Kindly please remove this image from here for the icons and try to add the icon by using pseudo-elements in CSS over the p tag, because these icons are not part of the flow of the HTML document and simply just for decoration so that's why please always try to add all fancy stuff especially decoration images or icon using CSS not in HTML.